language/stdlib/modules/transaction_fee.move

```diff
@@ -1,105 +1,87 @@
 address 0x0 {
 
 module TransactionFee {
-    use 0x0::LibraAccount;
-    use 0x0::LibraSystem;
     use 0x0::Signer;
     use 0x0::Transaction;
+    use 0x0::Coin1::T as Coin1;
+    use 0x0::Coin2::T as Coin2;
+    use 0x0::LBR::{Self, T as LBR};
+    use 0x0::Libra::{Self, T as Libra};
 
-    ///////////////////////////////////////////////////////////////////////////
-    // Transaction Fee Distribution
-    ///////////////////////////////////////////////////////////////////////////
-    // Implements a basic transaction fee distribution logic.
-    //
-    // We have made a couple design decisions here that are worth noting:
-    //  * We pay out once per-block for now.
-    //    TODO: Once we have a better on-chain representation of
-    //          epochs this should be changed over to be once per-epoch.
-    //  * Sometimes the number of validators does not evenly divide the transaction fees to be
-    //    distributed. In such cases the remainder ("dust") is left in the transaction fees pot and
-    //    these remaining fees will be included in the calculations for the transaction fee
-    //    distribution in the next epoch. This distribution strategy is meant to in part minimize the
-    //    benefit of being the first validator in the validator set.
+    resource struct FeePaymentCapability { }
 
-    resource struct TransactionFees {
-        fee_withdrawal_capability: LibraAccount::WithdrawalCapability,
+    resource struct CollectedFees<CoinType> {
+        preburn: Libra::Preburn<CoinType>,
```

Comment:
Might be useful to have a getter that allows you to read the quantity of collected fees.

Reply:
This can now be done by just looking at the balance for the currency under `0xFEE`.